### DISCUSSION TITLE
ECHIPA JMEKERILOR

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#110] [TEAMs]
repo: https://github.com/peviitor-ro/spider_web_scraper_jmeker

Boga Sebastian
role: JMeter scraper
discordID: Sebastian Boga

**Comment by sebiboga** [comment]
Medeea Boamfa
role: software tester
discord: Medeee735

**Reply by sebiboga** [reply]
2 x Manual Testers

**Reply by sebiboga** [reply]
1 x Automation Tester

**Comment by AdrianaCionca** [comment]
Georgiana Redenstein
role: Manual Tester
discord: Georgiana Redenstein

**Comment by AdrianaCionca** [comment]
Alexandru Barna
role: Manual Tester
discord: alexandro987

**Comment by AdrianaCionca** [comment]
Alexandra Cristea
role: Manual Tester
discord: alexandra.cristea.

**Comment by AdrianaCionca** [comment]
Diana Negru
role: Manual Tester
discord: daianna

**Comment by AdrianaCionca** [comment]
Carmen Tundrea
role: Manual Tester
discord: Carmen

**Comment by AdrianaCionca** [comment]
Gabriel Marica
role: Manual Tester
discord: Gabriel Marica

**Comment by sebiboga** [comment]
Dan Popescu
role: Manual Tester
discord:
